charts/images.yaml

```diff
@@ -8,7 +8,7 @@ images:
 - name: etcd
   sourceRepository: github.com/gardener/etcd-custom-image
   repository: europe-docker.pkg.dev/gardener-project/public/gardener/etcd
-  tag: "v3.4.26-3"
+  tag: "v3.4.26-7"
```

Comment:
etcd-custom-image:v3.4.26-7 includes https://github.com/gardener/etcd-custom-image/pull/46, which is required for this PR, since this PR also changes SSL cert paths, which needed to be adapted in custom-image bootstrap script.

I was surprised that druid master branch only has `etcd-custom-image:v3.4.26-3`. It hasn't been updated in a long time, which seems to be a miss from us as maintainers :( . Anyway, it's good that we updated the custom-image version in this PR atleast, which is necessary for it to work correctly.

Reply:
I guess it's because `etcd-custom-image` get released only just by merging 1 PR. Couple of PRs on etcd-custom-image got merged like this https://github.com/gardener/etcd-custom-image/pull/45 and https://github.com/gardener/etcd-custom-image/pull/43